### ISSUE TITLE
Manually restore full backup made by pg_probackup on Windows

### ISSUE THREAD

**Issue by alexshakurin** [postgrespro/pg_probackup#78]
I created full backup by pg_probackup with --stream option. Now I'm trying to restore this backup by pg_probackup via restore command and manually by copying files to a new directory and I encountered the following issues:
1) When restoring with restore command everything works except that recovery.conf file is not generated. Is it OK?
2) But I have issues when I manually copy files from backup to another directory and start Postgres server. Server starts but then immediately prints errors to a log file. Error is "ERROR:  invalid page in block 0 of relation global/1262". This happens regardless of whether I create recovery.conf file or not. Is there a way to restore backup manually?

**Comment by gsmol** [comment]
Hello!

1. Yes, `--stream` flag produce autonomous backup, which **by default** restores on it`s own consistency point, relying on WAL files from pg_wal/pg_xlog using crash recovery mechanism just like pg_basebackup do.
But using any of the `--recovery-*` options or `--restore-as-replica` flag when restoring autonomous backup will automatically create recovery.conf with corresponding directives.

2. Manual copying currently is not possible, pg_probackup add its own small header to every page. This header is used for storing page actual position and compressed size. Granted, for FULL backup this header is not really needed, so we are planning to get rid of it in FULL mode soon. 

P.S. What forced you to resort to manual copying ?

**Comment by alexshakurin** [comment]
Thanks for you help! 
I was looking for the quickest way to restore copy of production database for development purposes without having to copy the whole backup directory. And as far as I can see I can just copy the latest created directory and run `pg_probackup restore` from it.

**Comment by gsmol** [comment]
Well, it is possible to just copy only latest backup from backup_catalig and then restore it. But it is not very convenient.

We are working on adding backup/restore via ssh on Windows. It will allow you to backup remote database and restore backup on remote machine. Will it resolve your problem?

**Comment by alexshakurin** [comment]
Yes, that would be very helpful. Looking forward to see it in action. Thanks!

**Comment by gsmol** [comment]
What Windows version do you use?

**Comment by alexshakurin** [comment]
I use Windows 10 x64
